web/client/tests/unit/test_report_converter.py

```diff
@@ -0,0 +1,77 @@
+# -------------------------------------------------------------------------
+#
+#  Part of the CodeChecker project, under the Apache License v2.0 with
+#  LLVM Exceptions. See LICENSE for license information.
+#  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# -------------------------------------------------------------------------
+"""
+Test the report converter between the common Report type and the
+thrift ReportData type
+"""
+
+import unittest
+
+from codechecker_common import report
+from codechecker_client import report_type_converter
+from codechecker_api.codeCheckerDBAccess_v6 import ttypes
+
+
+class ReportTypeConverterTest(unittest.TestCase):
+    """Type conversion tests."""
+
+    def test_Report_to_ReportData(self):
+        """Report to reportData conversion."""
+        check_name = "checker.name"
+        report_hash = "2343we23"
+        source_file = "main.cpp"
+        description = "some checker message"
+        line = 10
+        column = 8
+
+        main = {
+            "description": description,
+            "check_name": check_name,
+            "issue_hash_content_of_line_in_context": report_hash,
+            "location": {"line": line, "col": column, "file": source_file},
+        }
+
+        rep = report.Report(main=main, bugpath=[], files={}, metadata=None)
+        severity_map = {check_name: "LOW"}
+        rep_data = report_type_converter.report_to_reportData(rep, severity_map)
+
+        self.assertEqual(rep_data.checkerId, rep.check_name)
+        self.assertEqual(rep_data.bugHash, rep.report_hash)
+        self.assertEqual(rep_data.checkedFile, rep.file_path)
+        self.assertEqual(rep_data.line, rep.line)
+        self.assertEqual(rep_data.column, rep.col)
+        # The severity level number for LOW is 20
+        self.assertEqual(rep_data.severity, 20)
```

Comment:
```suggestion
        self.assertEqual(rep_data.severity, ttypes.Severity.LOW)
```
And the comment can be removed above this line.